core/templates/dev/head/services/ExtensionTagAssemblerService.ts

```diff
@@ -18,6 +18,7 @@
  */
 
 require('filters/string-utility-filters/camel-case-to-hyphens.filter.ts');
+require('services/HtmlEscaperService.ts');
```

Comment:
Why is this required?

Reply:
I've added the require statement that was missing in this file. See line 32 of the file, the required service is being used.